### PR TITLE
[JS] Topic: Tutorial for JavaScript Operators

### LINKED ISSUE #202: JS: Operators in JavaScript
# Operators in JavaScript

### Directory

```
Topics/5.Syntax/2.Operators
```

### Description

* Talk about the various operators in JavaScript.
* Arithmetic, Assignment, Increment and Decrement, Comparison, Logical, Bitwise.
* Explain operators types through examples.

### Hints

* Check out the [W3C tutorial](https://www.w3schools.com/js/js_operators.asp)

## Formats
- [x] Document
- [ ] Video
- [ ] Audio

### PR DESCRIPTION
**Pull request by Himanshu664**
<hr>

## Description 📜

Created a tutorial for JavaScript Operators.
Fixes #202

<hr>

## Type of change 📝

- [ ] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev (Flutter, Java, Kotlin)
- [ ] Backend Dev (Java, .NET, Python)
- [ ] C/CPP
- [ ] Competitive Programming
- [ ] Cyber Security
- [ ] DSA
- [ ] Database
- [ ] Datascience with Python
- [ ] Datascience with R
- [ ] Frontend Dev (HTML, CSS, JS)
- [ ] Frontend Dev (React, Angular, Vue)
- [ ] Golang
- [ ] Interview Prep
- [ ] Java
- [x] JavaScript
- [ ] MERN
- [ ] Machine Learning
- [ ] Open Source
- [ ] Python
- [ ] Rust
- [ ] Statistics
- [ ] UI/UX

<hr>
 
## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Himanshu664** [comment]
@paramsiddharth please review this PR.

**Review by paramsiddharth — COMMENTED** [review]
Please describe the addition of the submodule. The rest looks good to me.

**Comment by Himanshu664** [comment]
@paramsiddharth I have made changes as suggested please review.

**Comment by paramsiddharth** [comment]
@Creatoon @dakshsethi Please review.

File: Web_Development/Javascript/Topics/5.Syntax/2.Operators/README.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+
+* **Addition(+)** - It adds the numerical operands and is also used for string concatenation.It will add numerical and string operand to a number if the string can be converted to a number. Else it concatenates them.
+
+    **Example:**  
+    ```js
+                  var x = 10;      // assign the value 10 to x
+                  var y = 4;       // assign the value 4 to y
+                   alert(x + y);    // 0utput: 14
+    ```
```
Comment (paramsiddharth) [inline Web_Development/Javascript/Topics/5.Syntax/2.Operators/README.md:19]: You don't need to indent the code-blocks!

````
    ```js
    var x = 10;      // assign the value 10 to x
    var y = 4;       // assign the value 4 to y
    alert(x + y);    // 0utput: 14
    ```                     
````

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
Please get rid of the indentation from all code-blocks. The rest seems great!

Also, please talk about the other and newly added operators, including the ternary operator, null-coalescing, et cetera.

**Comment by Himanshu664** [comment]
@paramsiddharth as you suggested I removed indent the code-blocks and also added new operators like ternary operator, null-coalescing.please merge this PR I dont want to create new PR because I am confused as the path is changed.

**Pull request closed without merging**